test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/DateSearchTests.cs

```diff
@@ -214,5 +214,27 @@ public async Task GivenAnOutOfRangeDateTimeSearchParam_WhenSearched_ThenExceptio
                 fce.StatusCode == HttpStatusCode.BadRequest,
                 $"A '{nameof(FhirClientException)}' with '{HttpStatusCode.BadRequest}' status code was expected, but instead a '{nameof(FhirClientException)}' with '{fce.StatusCode}' status code was raised. Url: {Client.HttpClient.BaseAddress}. Activity Id: {fce.Response.GetRequestId()}. Error: {fce.Message}");
         }
+
+        [Theory]
+        [InlineData("1981-05-16T16:32:15.500", 7)] // This should include the observation containing a period since the date is in that effective period.
```

Comment:
This test shouldn't pass. The search is for 1981 but the data is in 1980.

Reply:
Good catch, that was a typo on my part. Fixed.